src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -10,11 +10,19 @@
  */
 public class StringUtil {
     public static boolean containsIgnoreCase(String source, String query) {
-        String[] split = source.toLowerCase().split("\\s+");
-        List<String> strings = Arrays.asList(split);
+        List<String> strings = listifyString(source);
```

Comment:
Simply abstracting out a reusable method. I dont see how that breaks slap, unless ud want to wrap the remaining lines of code in the body into another method unecessarily.